sycl/include/CL/sycl/access/access.hpp

```diff
@@ -58,6 +58,7 @@ enum class address_space : int {
   generic_space = 6, // TODO generic_space address space is not supported yet
 };
 
+enum class decorated { no = 0, yes = 1, legacy = 2 };
```

Comment:
```suggestion
enum class decorated : int { no = 0, yes = 1, legacy = 2 };
```
this is how it is defined in spec.

Reply:
Sorry, I've missed this.